bftengine/src/bftengine/SigManager.cpp

```diff
@@ -237,12 +252,10 @@ bool SigManager::verifySig(
   return result;
 }
 
-void SigManager::sign(const char* data, size_t dataLength, char* outSig, uint16_t outSigLength) const {
+void SigManager::sign(const char* data, size_t dataLength, char* outSig) const {
```

Comment:
What if caller calls sign without allocating buffer signature. In such a case, code will crash. Ideally with an out buffer, we should also pass the length which tells the size of outSig buffer. And we can validate if buffer is large enough to hold signature before performing memcpy